File: skills/astra-toolkit/clients/go/examples/client-get-astra-admin.go
package main

import (
	"log"
	"os"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	client := astra.NewClient(
		options.API().SetToken(os.Getenv("ASTRA_DB_APPLICATION_TOKEN")),
	)

	_, err := client.Admin(options.API())

	if err != nil {
		log.Fatal(err)
	}
}

File: skills/astra-toolkit/clients/csharp/examples/client-get-astra-admin.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static void Main()
  {
    var client = new DataAPIClient(System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN"));

    var admin = client.GetAstraDatabasesAdmin();
  }
}
